website/docs/reference/runtimes/prisma.mdx

```diff
@@ -122,6 +122,10 @@ A one-to-many relationship must be in one of these two variants.
 | 1..1 ↔ 0..n | `g("Model2")`            | `t.array(g("Model1"))` |
 | 0..1 ↔ 0..n | `g("Model2").optional()` | `t.array(g("Model1"))` |
 
+### Many-to-many relationships
```

Comment:
I think the modification should be on the doc comment, otherwise this would be overriden.

Reply:
I moved out the documentation from the code to docusaurus in anticipation of the core SDK.